README.md

```diff
@@ -1,95 +1,108 @@
-- **Only \*gosub_scene dependencies**
-Public cslib functions can depend on (call) one another (for e.g. string:find uses string:substring) but they **must** use \*gosub_scene and **not** \*gosub, even if they reside within the same scene. Yes, this is horrible for performance, but it guarantees correct scoping (preventing different function's variables overwriting one another) and removes a whole category of nasty bugs for end-users.
-- **cslib_ret must be set**
-All cslib functions **must** \*set cslib_ret to a value before they \*return. If the function doesn't need to return a value, it should still set cslib_ret to an empty string: "". This will help discourage over-reliance on cslib_ret, which could lead to nasty bugs, or code that is difficult to read.
-- **config variables**
-Adding any new global config variables to cslib is invasive and increases  configuration effort. Adding one should be considered a breaking change, and only ever be done as a last resort.
--  **Use \*bug for unrecoverable errors**
-Any unrecoverable error state in a function should be handled by \*bug, and contain as much useful info as possible: file name, function name, variable values etc. It should not try to print the error message to the game screen.
-- **Tests**
-All cslib functions should have tests covering at least the basic use-cases. If the function is something that isn't easily tested (like a lot of cslib:menu), try to break it down into smaller (internal) functions that can be.
-- **Comments**
-All public cslib functions should be self-documenting, i.e. contain \*comment lines before them that briefly describe: the purpose of the function, its arguments and its return value (if any).
+*label flavour_text_strength
 
+The highest stat is "strength"! Amazing! 
+```
 
-#### Naming
+### Modules
 
-Below follows guidance on naming variables and routines etc., when contributing code to cslib.
+|    Module    |               Description               | Version | Depends on |                          Routines                          |
+|:------------:|:---------------------------------------:|:-------:|:----------:|:----------------------------------------------------------:|
+| cslib_string | Routines to manipulate text (strings).  |  1.0.0  |            | concat, lowercase, substring, find, replace, reverse, etc. |
+| cslib_number |     Routines to manipulate numbers.     |  1.0.0  |            |                    mean, max_stat, max, min, etc.                    |
+|  cslib_menu  | Routines to generate menus and choices. |  1.0.0  |           |                                                            |
+|  cslib_loop  |   Repeat the execution of a routine.    |  1.0.0  |            |                           repeat                           |
```

Comment:
CJW is adding cslib_array

Reply:
Depending on the merge order, I can add this as part of the cslib_array patch. I guess merge order depends on @danspinola though :)

Reply:
I'll make this adjustments and include the cslib_array. 😊